packages/server/tests/acceptance/rpc.spec.ts

```diff
@@ -96,6 +97,125 @@ describe('RPC Server Acceptance Tests', function () {
             mirrorSecondaryAccount = (await mirrorNode.get(`accounts?account.id=${accounts[1].accountId}`)).accounts[0];
         });
 
+        describe('eth_getLogs', () => {
+
+            let log0Block, log4Block, contractAddress;
+
+            it('should deploy a contract', async () => {
+                const logsContract = await servicesNode.deployContract(logsContractJson);
+                const mirrorNodeResp = await mirrorNode.get(`/contracts/${logsContract.contractId}`);
+                expect(mirrorNodeResp).to.have.property('evm_address');
+                expect(mirrorNodeResp.env_address).to.not.be.null;
+                contractAddress = mirrorNodeResp.evm_address;
+
+                const params = new ContractFunctionParameters().addUint256(1);
+                const log0 = await accounts[1].client.executeContractCall(logsContract.contractId, 'log0', params);
+                await accounts[1].client.executeContractCall(logsContract.contractId, 'log1', params);
+
+                params.addUint256(1);
+                await accounts[1].client.executeContractCall(logsContract.contractId, 'log2', params);
+
+                params.addUint256(1);
+                await accounts[1].client.executeContractCall(logsContract.contractId, 'log3', params);
+
+                params.addUint256(1);
+                const log4 = await accounts[1].client.executeContractCall(logsContract.contractId, 'log4', params);
+
+                await new Promise(r => setTimeout(r, 5000));
+
+                const logs = await relay.call('eth_getLogs', [{}]);
+                expect(logs.length).to.be.greaterThan(0);
+
+                log0Block = await relay.call('eth_getTransactionByHash', [log0.contractExecutedTransactionId]);
+                expect(log0Block).to.have.property('blockNumber');
+
```

Comment:
`log.address` and `log.logIndex` should also be checked for now.
Seems like we also have the blockNumber and blockHash so we should verify those here and below where possible also.
We can expand further in future

Reply:
I think you covered logIndex but not sure you covered the value of `address`